docs/src/3-prestashop-account/README.md

```diff
@@ -14,127 +14,75 @@ The backend of a module integrated with PrestaShop Cloud Services is made of [PH
 
 ## Backend
 
-### Composer
-
-Make sure [Composer](https://getcomposer.org/) is installed on your machine.
-
-At the root of your module folder, create a `composer.json` file with the following contents:
-
-```json{13,14}
-{
-    "name": "prestashop/<module_name>",
-    "description": "",
-    "config": {
-        "preferred-install": "dist",
-        "optimize-autoloader": true,
-        "prepend-autoloader": false
-    },
-    "require-dev": {
-        "prestashop/php-dev-tools": "^4.2.1"
-    },
-    "require": {
-        "php": ">=5.6",
-        "prestashop/prestashop-accounts-installer": "^1.0.1",
-        "prestashop/module-lib-service-container": "^1.4"
-    },
-    "autoload": {
-        "classmap": [
-            "<module_name>.php"
-        ]
-    },
-    "author": "PrestaShop",
-    "license": "MIT"
-}
-```
-
-### Register Your Module and PrestaShop Account as Services
-
-First, you need to register your module and PrestaShop Account as services. 
-
-:::tip Note
-For more information about services, see the [related documentation](https://devdocs.prestashop.com/1.7/modules/concepts/services/).
-:::
-
-At the root of your module folder, create a `config` folder containing a `common.yml` file with the following contents:
-
-```yaml
-services:
-  _defaults:
-    public: true
-
-  ##############
-  # Your Module
-
-  <module_name>.module:
-    class: <module_name>
-    factory: ['Module', 'getInstanceByName']
-    arguments:
-      - '<module_name>'
-
-  <module_name>.context:
-    class: Context
-    factory: ['Context', 'getContext']
-
-  #####################
-  # PrestaShop Account
-  ps_accounts.installer:
-    class: 'PrestaShop\PsAccountsInstaller\Installer\Installer'
-    arguments:
```

Comment:
Here, you need to take the code from the sample module (https://github.com/PrestaShopCorp/partner-devtools.prestashop.com/blob/main/modules/rbm_example/config/common.yml) with public: true